### ISSUE TITLE
Typescript: (selectors) TS2717: Subsequent property declarations must have the same type.

### ISSUE THREAD

**Issue by Izhaki** [rematch/rematch#514]
```
node_modules/@rematch/select/src/typings.d.ts:77:5 - 
error TS2717: Subsequent property declarations must have the same type.  
Property 'selectors' must be of type '{ [key: string]: (state: SS, ...args: any[]) =>any; }', 
but here has type 'ModelSelectorsConfig<S>'.

77     selectors?: ModelSelectorsConfig<S>;
```

I believe it is a collide between [core/typings.d.ts](https://github.com/rematch/rematch/blob/1.0.0-beta.5/src/typings.d.ts#L160) and [select/typings.d.ts](https://github.com/rematch/rematch/blob/master/plugins/select/src/typings.d.ts#L70).

**Comment by mironal** [comment]
I have same problem.

**Comment by ShMcK** [comment]
seems like a typings issue with the select plugin.

Relates to a limitation in typescript: it won't let you inherit typings from another package. Select builds upon the core typings, and can build from the same directory and will run locally, but allow you to inherit typings from another module.

Any thoughts on how to resolve this?

**Comment by d3dc** [comment]
When I committed my changes, I removed the select typings from core's `/src/typings.d.ts`. There hasn't been a core release since then, right?

The only thing I can foresee is a type error when you don't have a plugin installed - which is desirable.

**Comment by d3dc** [comment]
This particular typing issue was fixed after the `1.0.0` release.
